release-please-config.json

```diff
@@ -1,8 +1,24 @@
 {
-  "initial-version": "0.1.0",
+  "bootstrap-sha": "2bb9ae256af30ce27837c71b34da0705604186a3",
   "packages": {
-    ".": {},
-    "tools": {}
+    "gax": {
+      "changelog-path": "CHANGELOG.md",
+      "release-type": "node",
+      "bump-minor-pre-major": false,
+      "bump-patch-for-minor-pre-major": false,
+      "draft": false,
+      "prerelease": false
+    },
+    "tools": {
+      "changelog-path": "CHANGELOG.md",
+      "release-type": "node",
+      "bump-minor-pre-major": false,
+      "bump-patch-for-minor-pre-major": false,
+      "draft": false,
+      "prerelease": false
+    },
+    ".": {}
```

Comment:
I don't think you need/want a root component here.

Reply:
Will be fixed by: https://github.com/googleapis/synthtool/pull/1860